src/pcd/tests.rs

```diff
@@ -57,6 +57,7 @@ fn get_shard_start_end_values(
 }
 
 #[test]
+#[ignore]
```

Comment:
its not up to spec with the current zkEngine API (I think it still works though lol),  just discouraging anyone from testing it for now... It will get up to spec now/soon with us doing the aggregation stuff

Reply:
kinda random thing to do since it does not relate to the fix, but just felt like ignoring it

Reply:
It will get updated soon with the verifier circuit feature (`pcd` folder API should remain relatively the same)

Reply:
just checked now, test still works... in case you were worried :sweat_smile:  (rightfully so)

Reply:
Good to know. All of the integration tests use the mock backend, meaning if this would've been broken, it wouldn't have been caught automatically. We probably need to add more tests for novanet as well but with a WASM that doesn't destroy the CI.